cmd/dwh/main.go

```diff
@@ -49,7 +49,7 @@ func run() {
 	log.G(ctx).Info("starting DWH service", zap.String("grpc_addr", cfg.GRPCListenAddr),
 		zap.String("http_addr", cfg.HTTPListenAddr))
 	if err := w.Serve(); err != nil {
-		log.G(ctx).Error("cannot start DWH service", zap.Error(err))
+		log.G(ctx).Error("DWH failed", zap.Error(err))
```

Comment:
It will be smth like "DWH failed - context canceled" during shutdown, won't it?

Reply:
"context cancelled" is already present in the error itself:

```
// Canceled is the error returned by Context.Err when the context is canceled.
var Canceled = errors.New("context canceled")
```